compiler/src/dotty/tools/dotc/transform/MacroTransformWithImplicits.scala

```diff
@@ -0,0 +1,90 @@
+package dotty.tools.dotc
+package transform
+
+import core._
+import typer._
+import ast.Trees._
+import Contexts._
+import Symbols._
+import Decorators._
+import collection.mutable
+import annotation.tailrec
+
+/** A Macrotransform that maintains the necessary infrastructore to support
+ *  contxtual implicit searches (type-scope implicits are supported anyway).
+ */
+abstract class MacroTransformWithImplicits extends MacroTransform {
```

Comment:
Should this really go in if we do not use it? What are the other use cases where it might be needed?

Reply:
someone might want to fix the typos in the doc-comment, too...